PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/org/jetbrains/plugins/azure/functions/run/AzureFunctionsIsolatedDebugProfile.kt

```diff
@@ -80,7 +79,6 @@ class AzureFunctionsIsolatedDebugProfile(
         private const val DOTNET_ISOLATED_DEBUG_ARGUMENT = "--dotnet-isolated-debug"
         private const val DOTNET_ENABLE_JSON_OUTPUT_ARGUMENT = "--enable-json-output"
         private const val DOTNET_JSON_OUTPUT_FILE_ARGUMENT = "--json-output-file"
-        private val controlCharsRegex = "\u001B\\[[\\d;]*[^\\d;]".toRegex()
     }
 
     private var processId = 0
```

Comment:
The only change I'd recommend here is to add `@Volatile`.

Aside from that, you may consider upgrading this to be `Deferred<Int>` instead of just `Int`, but that's optional.

Reply:
I removed access from different threads, so it looks like `@Volatile` is not longer needed